README.md

```diff
@@ -103,6 +103,7 @@ Requirements for the lib to work are:
 ## Segment Requirements
 Some segments have their own requirements. If you enable them in your theme, make sure all requirements are met for those.
 
+* **cpu_temp.sh**: `lm_sensors` for Linux, `smctemp` for Macos
```

Comment:
How can `smctemp` be installed on macOS? `brew search` did not find a match on `smctemp`.